app/controllers/keywords_controller.rb

```diff
@@ -11,8 +11,22 @@ def index
     }
   end
 
+  def show
+    keyword = Keyword.find_by(id: params[:id])
+
+    if keyword.present?
+      render locals: {
+        keyword: keyword
+      }
+    else
+      render :not_found, status: :not_found
+    end
+  end
+
   def create
     if @csv_import_form.save(create_params)
+      GoogleScrapingJobManagementJob.perform_later(@csv_import_form.inserted_keywords.rows)
```

Comment:
Can we use [ActiveRecord callback](https://guides.rubyonrails.org/active_record_callbacks.html) with `insert bulk` 🤔 

Normally we create the record 1 by 1 so the ActiveRecord callback can be applied in this case

Reply:
We can't use `ActiveRecord callback` because `insert_all` doesn't trigger callbacks [(Ref)](https://edgeguides.rubyonrails.org/active_record_callbacks.html#running-callbacksl)